src/tomography.jl

```diff
@@ -0,0 +1,244 @@
+using Shortcuts, SCS, QIP, Convex
```

Comment:
Also, @marcusps says that QIP is deprecated, and that we should now use QuantumInfo.